### PR TITLE
[IMP] ecommerce: updated variants page

### PR DESCRIPTION
**Pull request by toaa-odoo**
Previous page was mostly on the sales app. New page focuses on ecommerce with :ref: to sales. Forward to master.

taskid: 3121379

### CODE REVIEW

File: content/applications/websites/ecommerce/managing_products/variants.rst [hunk 1]
```diff
@@ -1,110 +1,88 @@
...
-
-Then, select a product from the Products list (:menuselection:`Sales / Website --> Products -->
-Products`). When the product detail form opens, click on *Edit*, and go to the *Variants* tab.
-
-Here, you can add as many different attributes as you'd like by clicking on *Add a line*.
-
-These attributes appear as three different types:
+.. note::
+   :guilabel:`Product Variants` has to be enabled under :menuselection:`Configuration --> Settings`,
+   in the :guilabel:`Shop - Products` section.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/managing_products/variants.rst:12]:
```suggestion
   :guilabel:`Product Variants` have to be enabled under :menuselection:`Configuration --> Settings`,
   in the :guilabel:`Shop - Products` section.
```

I know it's "the option" that has to be enabled, but it sounds weird when the words before the verb are in the plural form

File: content/applications/websites/ecommerce/managing_products/variants.rst [hunk 2]
```diff
@@ -1,110 +1,88 @@
...
 
-Several variants appear as soon as there are two values for one attribute.
+Product configurator
+====================
 
-Don't forget to hit *Save* once all variants have been entered.
+In the context of eCommerce, adding attributes and values to a product template allows for the
+addition of a **product configurator** on the product page. The customer is able to configure and
+select the product variant of their choice; or in the case of multiple attributes, combine those to
+create a specific variant.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/managing_products/variants.rst:24]:
```suggestion
In the context of eCommerce, adding attributes and values to a product template allows for adding a **product configurator** on the product page. Customers are able to configure and
select the product variant of their choice; or in the case of multiple attributes, combine those to
create a specific variant.
```
I would change it to a plural as "the customer" (sg) --> of "his/her" choice

File: content/applications/websites/ecommerce/managing_products/variants.rst [hunk 3]
```diff
@@ -1,110 +1,88 @@
...
+line**.
 
-.. note::
-   Both the Barcode and the Internal Reference are variant-specific. You need to populate them once
-   the variants are generated.
+.. tip::
+   Specific combinations of values can be excluded from the product configurator. The customers are
+   then not able to select the excluded combination of values. To do so, access the
+   :ref:`Configure <variants-configurator-backend>` menu, select a **value**, and fill in the
+   :guilabel:`Exclude for` line.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/managing_products/variants.rst:53]:
```suggestion
   Specific combinations of values can be excluded from the product configurator. This way, customers are
  unable to select the excluded combination of values. To do so, access the
   :ref:`Configure <variants-configurator-backend>` menu, select a **value**, and fill in the
   :guilabel:`Exclude for` line.
```

**Review by jaae-odoo — APPROVED** [review]
Good doc, everything is clear. I only made some suggestions for the grammar, but up to you if you want to change them or not.

File: content/applications/websites/ecommerce/managing_products/variants.rst (hunk 2 above)
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/managing_products/variants.rst:24]: I changed it up a bit, thanks for the suggestion! FYI, "their" is the neutral form (and therefore preferred), and works for both Sg. and Pl. :)

File: content/applications/websites/ecommerce/managing_products/variants.rst [hunk 4]
```diff
@@ -1,110 +1,88 @@
...
 
-You can disable/archive specific variants so they are no longer available in quotes or on your
-website (not existing in your stock, deprecated, etc.). Simply select *Archive* in their detail
-form. You can reactivate them the same way.
+To enable **attributes filtering**, go to :menuselection:`Edit --> Customize` from your **main shop
+page** and click on one of the **categories** in the left column. Here, enable either
+:guilabel:`Left`, :guilabel:`Top`, or **both**, in the :guilabel:`Attributes` field.
 
-.. image:: variants/archive.png
+.. image:: variants/variants-catalog.png
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/managing_products/variants.rst:82]:
```suggestion
.. image:: variants/variants-catalog.png
```
Not really useful IMO - you don't really know what you are looking at.
If you really want a screenshot here, maybe show the categories option which is a bit difficult to find.
image

File: content/applications/websites/ecommerce/managing_products/variants.rst [hunk 5]
```diff
@@ -1,110 +1,88 @@
...
    :align: center
-   :alt: ecommerce product variants archive
-
-To retrieve such items, hit *Archived* in the *Filters* category while searching the variants list.
+   :alt: Variants catalog filtering
 
-.. image:: variants/archive-search.png
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/managing_products/variants.rst:105]: Not useful, especially without much context

**Review by xpl-odoo — CHANGES_REQUESTED** [review]

**Review by xpl-odoo — APPROVED** [review]

**Comment by toaa-odoo** [comment]
Do not merge. Part of mega PR.

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**